src/components/CategoriesView.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const CategoriesView = () => (
+  <section className="container mx-auto">Categories</section>
+);
```

Comment:
- [ ] According to the project requirements, there should be a check status button rendered on this page as seen below 👍 
![image](https://user-images.githubusercontent.com/45460192/177829091-687c50e2-f1d1-47f1-8536-b6b4b9977767.png)